remote-api/src/main/java/org/alfresco/rest/api/model/rules/SimpleCondition.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.rest.api.model.rules;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.alfresco.repo.action.evaluator.CompareMimeTypeEvaluator;
+import org.alfresco.repo.action.evaluator.ComparePropertyValueEvaluator;
+import org.alfresco.repo.action.evaluator.HasAspectEvaluator;
+import org.alfresco.repo.action.evaluator.HasChildEvaluator;
+import org.alfresco.repo.action.evaluator.HasTagEvaluator;
+import org.alfresco.repo.action.evaluator.HasVersionHistoryEvaluator;
+import org.alfresco.repo.action.evaluator.InCategoryEvaluator;
+import org.alfresco.repo.action.evaluator.IsSubTypeEvaluator;
+import org.alfresco.repo.action.evaluator.NoConditionEvaluator;
+import org.alfresco.service.Experimental;
+import org.alfresco.service.cmr.action.ActionCondition;
+import org.apache.commons.collections.CollectionUtils;
+
+@Experimental
+public class SimpleCondition
+{
+    private static final String COMPARATOR_EQUALS = "equals";
```

Comment:
Are you planning another PR for other comparators?

Reply:
Generally comparators/operators are placed in parameterValue map field `.PARAM_OPERATION`. This "equals" comparator is used only for cases like: "Has aspect: audio", or "Mimetype is 3GVideo", where `.PARAM_OPERATION` is missing in Map.